proxy/src/main/java/org/dragonet/proxy/protocol/packets/StartGamePacket.java

```diff
@@ -83,7 +83,7 @@ public void encodePayload() {
         putBoolean(multiplayerGame);
         putBoolean(lanBroadcast);
         putBoolean(xboxLiveBroadcast);
-        putBoolean(commandsEnabled);
+        putBoolean(true);
```

Comment:
I don't like this. It's hacky.

Reply:
Otherwise it's always set to false.  Having it on false prevents any commands from getting through to the server (prevents the client from sending them), whereas in PC the command is sent as a plain chat message and left to the server to parse.  We have to assume that the server will accept commands, and will send an error message itself in the case that it won't.